src/Uno.UI/UI/Xaml/Controls/Control/Control.Engagement.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Windows.Foundation;
+using Uno;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class Control
+	{
+		public event TypedEventHandler<Control, FocusEngagedEventArgs> FocusEngaged
```

Comment:
Agree for the `[NotImplemented]`, I've added it ... and I also agree for the file, @carldebilly why you added this file instead instead of just keeping the generated code?